testsite/myapp/settings.py

```diff
@@ -115,7 +116,8 @@
 )
 
 CIVET_SASS_ARGUMENTS = ('--compass', )
-CIVET_IGNORE = ['bower_components', 'to_be_ignored']
+CIVET_IGNORE_PATTERNS = ['']
```

Comment:
If this is not used, you can just remove that settings, and have Civet fall back to its default.